### PR TITLE
Tags Selector 2.0

### PR DESCRIPTION
**Pull request by MengZhuNAV**
updated tags selector and teamkatalogen selector

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -15,6 +15,6 @@
     "jsx": "preserve",
     "incremental": true
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "components/lib/useFormField.js"],
```
Comment (chinatsu) [inline tsconfig.json:18]: `"components/lib/useFormField.js"` is not a file that exists as far as i can tell?

File: components/lib/tagsSelector.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  useKeywordsQuery,
+  useTeamkatalogenQuery,
+} from '../../lib/schema/graphql'
+import ErrorMessage from './error'
+import LoaderSpinner from './spinner'
+import { Dispatch, SetStateAction, useContext, useEffect } from 'react'
+import { Dropdown } from '@navikt/ds-react-internal'
+import CreatableSelect from 'react-select/creatable'
+import { CONTINUOUS_DOMAIN_SCALES } from 'vega-lite/build/src/scale'
+import classNames from 'classnames'
```
Comment (chinatsu) [inline components/lib/tagsSelector.tsx:13]: this import isn't used as far as i can tell

**Comment by chinatsu** [comment]
also general question: the styling changes you showed on slack, they're not in this branch (yet?)

File: tsconfig.json (hunk 1 above)
Comment (MengZhuNAV) [reply tsconfig.json:18]: no, we can remove it

File: components/lib/tagsSelector.tsx (hunk 2 above)
Comment (MengZhuNAV) [reply components/lib/tagsSelector.tsx:13]: true!

**Pull request merged**